city_scrapers/spiders/wayne_land_bank.py

```diff
@@ -0,0 +1,76 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from scrapy.http import FormRequest
+
+
+class WayneLandBankSpider(CityScrapersSpider):
+    name = "wayne_land_bank"
+    agency = "Wayne County Land Bank"
+    timezone = "America/Detroit"
+    allowed_domains = ["public-wclb.epropertyplus.com"]
+    start_urls = ["https://public-wclb.epropertyplus.com/landmgmtpub/app/base/customPage"]
+    location = {
+        "name": "Wayne County Treasurer's Office",
+        "address": "400 Monroe St 5th Floor, Detroit, MI 48226, United States",
+    }
+
+    def start_requests(self):
+        yield FormRequest(
+            self.start_urls[0],
+            formdata={
+                "templateName": "BoardofDirectors",
+                "title": "Board+of+Directors"
+            },
+            callback=self.parse,
+            dont_filter=True
+        )
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        data = response.xpath('//script[@id="custom_page_template"]/text()').extract()
+        data = re.findall(
+            '(Jan(?:uary)|Feb(?:ruary)|Mar(?:ch)|Apr(?:il)|May|Jun(?:e)|Jul(?:y)'
+            '|Aug(?:ust)|Sep(?:tember)|Oct(?:ober)|Nov(?:ember)|Dec(?:ember))'
+            r'(.*?)(?=\.\s|\\n)'
+            '(.*?)(?=\\n)', data[0]
+        )
+
+        for item in data:
+            meeting = Meeting(
+                title=item[2].strip(" .()"),
```

Comment:
The title should never be an empty string, if it's blank we should default to "Board of Directors". Since these are all for the Board, any additional title should probably be added to "Board of Directors " so that the special meeting becomes "Board of Directors Special Meeting"